### DISCUSSION TITLE
SelectItem withing Dialog not highlighted on hover

### DISCUSSION THREAD

**Discussion by OttlikG** [chakra-ui/chakra-ui#9298] [General] [answered]
### Description

When I highlight one item within Select component I expect to the hovered item gets highlighted. The data-highlighted attribute is put on even so I don't see it highlighted.

<img width="2140" alt="Screenshot 2024-12-06 at 9 30 19" src="https://github.com/user-attachments/assets/3ff55c4f-cf35-4e20-87a2-addcbb9e742e">

### Link to Reproduction

https://stackblitz.com/edit/chakra-ui-v3-1qqwp4?file=src%2FApp.tsx

### Steps to reproduce

1. Click on Open Dialog
2. For some reason it can't even render the Dialog with the Select component from the example.

At my local set up it renders just without the highlight.

### Chakra UI Version

latest

### Browser

Version 131.0.6778.86

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment, marked as answer]
I am moving this to discussion since I can't reproduce it locally. I used a fresh Chakra + Vite project, and here's the result.

https://github.com/user-attachments/assets/28741c0e-c0c8-49b4-a87d-1224244950d8

**Reply by segunadebayo** [reply]
You can consider upgrading to the latest chakra version

**Reply by OttlikG** [reply]
Hi Segun, thanks for replying back. I love using Chakra but I have a bit of headache while upgrading to v3. Do you know why the following [link](https://stackblitz.com/edit/chakra-ui-v3-1qqwp4?file=src%2FApp.tsx) doesn't display the Select component? It only display the dialog when the component is commented out. 
By the way I'm using NextJS could that give some extra complication while using v3? I followed the guideline of Next App router on Chakra v3 site.
I'm also using the @latest version of Chakra

**Reply by segunadebayo** [reply]
I wish we could make the migration easier. Sorry.

It also works in a Next.js app I tested, so I'm unsure where the issue originated.

**Reply by thesohailjafri** [reply]
@segunadebayo Man you have created an awesome ui framework. migration was much needed to boost chakra UI performance. Dont be sorry you did your best. I have been using Chakra UI for over 1 year and its my go to UI lib these days. Ludos to you and your team man!

**Comment by jdx** [comment]
Fixed in https://github.com/jdx/mise-versions/commit/39ce2ec1b6614281497b5c4ba216050d60d500e2 - the workflow now runs `curl https://mise.run | sh` to install the latest mise, overwriting the cached version from mise-action.
